backend/app/modules/projects/service/projects_metadata.py

```diff
@@ -0,0 +1,29 @@
+from flask import current_app as app
```

Comment:
I think that it would be better to change the dir structure a bit:
- change name of `project_rewards` to `projects`
- put `rewards` dir in it containing the rewards stuff
- add `metadata` dit and put your part there